Adds definition of time series data (#225)
elastic/docs-content: serverless/pages/manage-your-project.asciidoc

. The volume of non-time series project data
. The volume of time series project data included in the Search Boost Window

[NOTE]
====

> Time series data refers to any document in standard indices or data streams that include the `@timestamp` field. This field determines whether the data will be subject to the Search Boost Window setting.

Reviewer: ```suggestion
Time series data refers to any document in standard indices or data streams that includes the `@timestamp` field. This field must be present for data to be subject to the Search Boost Window setting.
```
